platform.md

```diff
@@ -786,20 +760,27 @@ Usage:
 ```
 
 ##### Inputs
-| Input               | Environment Variable  | Default Value          | Description
-|---------------------|-----------------------|------------------------|---------------------------------------
-| `<daemon>`          | `CNB_USE_DAEMON`      | `false`                | Export image to docker daemon
-| `<gid>`             | `CNB_GROUP_ID`        |                        | Primary GID of the build image `User`
-| `<image>`           |                       |                        | App image to rebase
-| `<log-level>`       | `CNB_LOG_LEVEL`       | `info`                 | Log Level
-| `<report>`          | `CNB_REPORT_PATH`     | `<layers>/report.toml` | Path to report (see [`report.toml`](#reporttoml-toml)
-| `<run-image>`       | `CNB_RUN_IMAGE`       | derived from `<image>` | Run image reference
-| `<uid>`             | `CNB_USER_ID`         |                        | UID of the build image `User`
+| Input         | Environment Variable | Default Value          | Description                                              |
+|---------------|----------------------|------------------------|----------------------------------------------------------|
+| `<daemon>`    | `CNB_USE_DAEMON`     | `false`                | Export image to docker daemon                            |
+| `<force>`     | `CNB_FORCE_REBASE`   | `false`                | Allow mismatched metadata between new and old run images |
+| `<gid>`       | `CNB_GROUP_ID`       |                        | Primary GID of the build image `User`                    |
+| `<image>`     |                      |                        | App image to rebase                                      |
+| `<log-level>` | `CNB_LOG_LEVEL`      | `info`                 | Log Level                                                |
+| `<report>`    | `CNB_REPORT_PATH`    | `<layers>/report.toml` | Path to report (see [`report.toml`](#reporttoml-toml)    |
+| `<run-image>` | `CNB_RUN_IMAGE`      | derived from `<image>` | Run image reference                                      |
+| `<uid>`       | `CNB_USER_ID`        |                        | UID of the build image `User`                            |
 
 - At least one `<image>` must be provided
 - Each `<image>` MUST be a valid tag reference
-- **If** `<daemon>` is `false` and more than one `<image>` is provided they MUST refer to the same registry
-- **If** `<run-image>` is not provided by the platform the value will be [resolved](#run-image-resolution) from the contents of the `stack` key in the `io.buildpacks.lifecycle.metdata` label on `<image>`.
+- **If** `<daemon>` is `false` and more than one `<image>` is provided, the images MUST refer to the same registry
+- **If** `<run-image>` is not provided by the platform, the value will be [resolved](#run-image-resolution) from the contents of the `stack` key in the `io.buildpacks.lifecycle.metdata` label on `<image>`
```

Comment:
We should probably change the key name - maybe something like `base`?

Similarly for stack.toml - maybe something like run.toml?

Suggestions are welcome...

Reply:
Or we could use this opportunity to refactor the `io.buildpacks.lifecycle.metadata` label, which I know @ekcasey has long wanted ;)

Reply:
Summarizing offline discussions: let's keep it as `stack` for now. The `io.buildpacks.lifecycle.metdata` is "user facing" but not as user facing as the `io.buildpacks.stack.*` labels.

We may make further changes when we do run image extensions.

Reply:
Maybe instead of:
```
  "runImage": {
    "topLayer": "<run-image-top-layer-diffID>",
    "reference": "<run-image-reference>"
  },
  "stack": {
    "runImage": {
      "image": "cnbs/sample-stack-run:bionic"
    }
  }
```
in the label, we could have
```
  "runImage": {
    "image": "cnbs/sample-stack-run:bionic",
    "mirrors": ["<run-image-mirror>"],
    "digest": "<run-image-reference>",
    "topLayer": "<run-image-top-layer-diffID>",
  },
```
and promote `run-image` and `build-image` to top-level keys in builder.toml.

Reply:
This is addressed in https://github.com/buildpacks/spec/pull/347